go/libraries/doltcore/table/map_point_reader.go

```diff
@@ -53,3 +53,8 @@ func (pr *PointReader) Next(ctx context.Context) (k, v types.Value, err error) {
 
 	return k, v, err
 }
+
+// NextTuple implements types.MapIterator.
+func (pr *PointReader) NextTuple(ctx context.Context) (k, v types.Tuple, err error) {
+	panic("not implemented")
```

Comment:
Should this return an error instead? Panics mean the code is busted...but maybe they're dangerous enough / likely enough in this case?